package.json

```diff
@@ -34,13 +34,13 @@
   "dependencies": {
     "chalk": "^2.4.1",
     "css": "^2.2.3",
-    "jest-diff": "^22.4.3",
-    "jest-matcher-utils": "^22.4.3",
-    "pretty-format": "^23.0.1",
+    "jest-diff": "^23.6.0",
+    "jest-matcher-utils": "^23.6.0",
+    "pretty-format": "^23.6.0",
```

Comment:
Not needed as it's within semver range (and `jest-diff` depends on it)m but doesn't hurt to keep them in sync